src/ROUTES.js

```diff
@@ -187,4 +187,8 @@ export default {
             isSubReportPageRoute: Boolean(lodashGet(pathSegments, 2)),
         };
     },
+
+    ROUTES_REGEX: {
+        UNLINK_LOGIN: /\/u($|(\/\/*))/,
+    },
```

Comment:
Is it okay to be here?  I think its better to be on CONST

Thoughts @francoisl?

Reply:
Yeah agreed. Looks like we even already have a `REGEX` object, which has a similar `ROUTES` sub-object [here](https://github.com/Expensify/App/blob/3324ea72cdabc84ba69e01bed8c56208a86b8e1f/src/CONST.js#L1180-L1182). That sounds like a good place to put this.